.gitignore

```diff
@@ -1,5 +1,5 @@
-dist/
```

Comment:
This needs to stay: https://github.com/blade-ui-kit/blade-zondicons/blob/main/bin/compile.sh#L6

Reply:
Fixed 👍